CONTRIBUTING.rst

```diff
@@ -28,9 +28,9 @@ on metatensor:
 Additionally, you will need to install the following software, but you should
 not have to interact with them directly:
 
-- **CMake**: we need a CMake version of at least 3.10.
-- **a C++ compiler** we need a compiler supporting C++11. GCC >= 5, clang >= 3.7
-  and MSVC >= 15 should all work, although MSVC is not yet tested continuously.
+- **CMake**: we need a CMake version of at least 3.16.
+- **a C++ compiler** we need a compiler supporting C++17. GCC >= 7, clang >= 5
```

Comment:
metatensor-core is still c++11, only metatensor-torch requires c++17: https://github.com/metatensor/metatensor/blob/main/metatensor-core/CMakeLists.txt#L373